pyproject.toml

```
@@ -48,17 +48,18 @@ redis = ["redis>=2.10.5"]
 # Development extras.
 dev = [
     "CacheControl[filecache,redis]",
-    "build",
     "cherrypy",
-    "codespell[tomli]",
+    # See: https://github.com/cherrypy/cherrypy/issues/2070
+    # See: https://github.com/cherrypy/cheroot/issues/769
+    "cheroot != 11.0.0",
```

It's more like
```suggestion
    "cheroot != 11.0.0, != 11.1.0, != 11.1.1",
```
so far.

Though, hopefully https://github.com/cherrypy/cheroot/pull/794 will get merged soon and I'll cut a new release right away.